### PR TITLE
README: Update Brewfile example

### PR DESCRIPTION
**Pull request by issyl0**
- Alphabetize the list of things, and put `tap` lines at the top.
- Don't include formulae options, as they have been deprecated for some
  time and won't work.
- Use a standard `homebrew/core` tap, not one from BitBucket?!
- Use homebrew/cask-fonts instead of caskroom/fonts.
- Add blank lines for readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
```
Comment (jacobbednarz) [inline README.md]: I wonder if this was here as an example of using a third party code host that didn’t follow the normal conventions and outside of GitHub?

File: README.md [hunk 2]
```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
```
Comment (MikeMcQuaid) [inline README.md]: This is a no-op effectively as it'll never be output by `brew bundle dump` and Homebrew taps it implicitly. I say remove it.

File: README.md [hunk 3]
```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
```
Comment (MikeMcQuaid) [inline README.md]: This needs to precede any other `cask` lines

File: README.md [hunk 4]
```diff
@@ -28,18 +28,20 @@ touch Brewfile
...
+tap "homebrew/core"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "homebrew/cask-fonts"
+
+brew "emacs"
 brew "imagemagick"
 brew "mysql@5.6", restart_service: true, link: true, conflicts_with: ["mysql"]
-brew "emacs", args: ["with-cocoa", "with-gnutls"]
```
Comment (MikeMcQuaid) [inline README.md]: Would be good to use an example from a third-party tap with options e.g. `denji/nginx/nginx-full --with-rtmp-module`

File: README.md [hunk 5]
```diff
@@ -28,18 +28,20 @@ touch Brewfile
...
+
 cask "google-chrome"
 cask "java" unless system "/usr/libexec/java_home --failfast"
 cask "firefox", args: { appdir: "~/my-apps/Applications" }
+cask "font-charter"
```
Comment (MikeMcQuaid) [inline README.md]: Use the full `"caskroom/fonts/font-charter"` format.

File: README.md (hunk 1 above)
Comment (MikeMcQuaid) [reply README.md]: Yep, it is. Would be good to keep something in this format.

File: README.md (hunk 4 above)
Comment (issyl0) [reply README.md]:
```suggestion
brew "denji/nginx/nginx-full", args: ["with-rmtp-module"]
```

File: README.md (hunk 5 above)
Comment (issyl0) [reply README.md]: Can this be “homebrew/cask-fonts/font-charter” instead?

File: README.md (hunk 3 above)
Comment (issyl0) [reply README.md]: This is interesting as I’ve never seen or needed this before. I’ll add it back.

File: README.md (hunk 4 above)
Comment (MikeMcQuaid) [reply README.md]: Looks good.

File: README.md (hunk 3 above)
Comment (MikeMcQuaid) [reply README.md]: Thanks!

File: README.md (hunk 5 above)
Comment (MikeMcQuaid) [reply README.md]: Yup!

File: README.md (hunk 1 above)
Comment (issyl0) [reply README.md]: Do you have an example of a commonly used formula in a non-standard tap (ie BitBucket or GitLab)?

**Comment by issyl0** [comment]
Thanks for the review, @MIkeMcQuaid, I've pushed changes.

File: README.md (hunk 1 above)
Comment (MikeMcQuaid) [reply README.md]: No, sorry 😭. I reckon the best bet is just a fake example that demonstrates the syntax but won't actually work. You can also remove `pin: true` as tap pinning is now deprecated.

Comment (issyl0) [reply README.md]: OK, I added a dummy example in bdca40a.

File: README.md [hunk 6]
```diff
@@ -28,18 +28,21 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "tap-from-non-standard-location", "https://user@bitbucket.org/user/homebrew-tap-repo.git"
```
Comment (MikeMcQuaid) [inline README.md]: Sorry to be pedantic but can this still be in a `user/repo` format e.g. `user/tap-repo`

File: README.md [hunk 7]
```diff
@@ -28,18 +28,21 @@ touch Brewfile
...
+cask_args appdir: "/Applications"
+
+brew "denji/nginx/nginx-full", args: ["with-rmtp-module"]
```
Comment (MikeMcQuaid) [inline README.md]: Put this one below the other examples so the first one is the simplest?

File: README.md [hunk 8]
```diff
@@ -28,18 +28,21 @@ touch Brewfile
...
+brew "emacs"
```
Comment (MikeMcQuaid) [inline README.md]: Can remove this as `imagemagick` below demonstrates this nicely.

File: README.md (hunk 7 above)
Comment (issyl0) [reply README.md]: I was going for alphabetical order, but sure.

File: README.md (hunk 6 above)
Comment (issyl0) [reply README.md]: Done!

**Comment by MikeMcQuaid** [comment]
Thanks @issyl0, you rock (as usual)!

**Pull request merged**